test/RelayedTransactionValidator.test.ts

```diff
@@ -0,0 +1,272 @@
+import { expect, use } from 'chai';
+import sinonChai from 'sinon-chai'
+import { createSandbox, SinonStubbedInstance } from 'sinon';
+import { RelayedTransactionValidator } from '../src/RelayedTransactionValidator';
+import ContractInteractor from '@rsksmart/rif-relay-common/dist/src/ContractInteractor';
+import chaiAsPromised from 'chai-as-promised';
+import type { DeployTransactionRequest, EnvelopingConfig } from '@rsksmart/rif-relay-common/dist/src';
```

Comment:
move the whole configuration including the `EnvelopingConfig` typing to the client please.